flathub.json

```
@@ -1,3 +1,4 @@
 {
-  "only-arches": ["x86_64"]
+  "only-arches": ["x86_64"],
+  "end-of-life": "This application has stopped development. See org.ferdium.Ferdium for a similar replacement."
```

```suggestion
  "end-of-life": "This application has stopped development."
  "end-of-life-rebase": "org.ferdium.Ferdium"
```

The maintenance section actually has a direct solution for this.